scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -97,14 +92,15 @@ import retrofit2.converter.jackson.JacksonConverterFactory
  * [NAMING_CONVENTION_VARIABLE_PREFIX] e.g. namingVariableVar1 = "foo".
```

Comment:
Should we not remove all the duplicated documentation to allow having to maintain documentation in two places ?

For instance we could put here `Options of the scanner including documentation is available in [FossIdConfig].`

Reply:
Moved the description of configuration options to the `FossIdConfig` class.